saleor/settings.py

```diff
@@ -384,11 +384,29 @@ def get_host():
 AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY')
 AWS_STORAGE_BUCKET_NAME = os.environ.get('AWS_STORAGE_BUCKET_NAME')
 
+# Google cloud storage
+GS_STORAGE_BUCKET_NAME = os.environ.get('GS_STORAGE_BUCKET_NAME')
+GS_MEDIA_BUCKET_NAME = os.environ.get('GS_MEDIA_BUCKET_NAME')
+
+# If GOOGLE_APPLICATION_CREDENTIALS is set with path to JSON credentials,
+# there is no need to load OAuth token
+# See https://django-storages.readthedocs.io/en/latest/backends/gcloud.html
+if not os.environ.get("GOOGLE_APPLICATION_CREDENTIALS"):
```

Comment:
Don't retrieve the value if you don't need it, just do the following instead:
```python
if 'GOOGLE_APPLICATION_CREDENTIALS' not in os.environ
```

Reply:
Totally right - Copy pasta to blame
Thanks!